src/ripple/app/ledger/impl/MetaView.cpp

```diff
@@ -543,11 +543,12 @@ Json::Value MetaView::getJson (int) const
 
 void
 MetaView::apply (BasicView& to,
-    STTx const& tx, TER ter, beast::Journal j)
+    std::shared_ptr<STTx const> const& tx,
```

Comment:
lol yeah a little bit I think - `MetaView` is never going to store them. Nor `ripple::apply`